### DISCUSSION TITLE
Customise/remove footnotes's heading

### DISCUSSION THREAD

**Discussion by bzrov** [remarkjs/remark#952] [Q&A] [answered]
Hello!
How can i customise or remove footnotes's heading with remark-gfm?

**Comment by wooorm** [comment, marked as answer]
There are options for that in remark-rehype, which takes care of turning markdown into html, as mentioned in the readme here: https://github.com/remarkjs/remark-gfm#when-should-i-use-this.

You can use CSS for more customization.

**Reply by bzrov** [reply]
Thank you!
But I can't find anything about turning off the heading in the documentation

**Reply by wooorm** [reply]
You can't turn it off. Why would you? You can do things with css.
